homeassistant/components/honeywell/__init__.py

```diff
@@ -163,7 +155,8 @@ async def _retry(self) -> bool:
             else:
                 _LOGGER.info(
                     "New device with ID %s detected, reload the honeywell integration"
-                    " if you want to access it in Home Assistant"
+                    " if you want to access it in Home Assistant",
+                    updated_device.deviceid,
```

Comment:
In another PR, can this be made more user friendly?

Reply:
Sure can - I was just adding the missing values to the debug string (that already existed).
When this is finished, I think HA should just add the devices "automagically", and this will go away.